libpgmodeler_ui/src/gqbcorewidget.h

```diff
@@ -0,0 +1,107 @@
+/*
+# PostgreSQL Database Modeler (pgModeler)
+#
+# Copyright 2006-2018 - Raphael Araújo e Silva <raphael@pgmodeler.io>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# The complete text of GPLv3 is at LICENSE file on source code root directory.
+# Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
+*/
+
+/**
+\ingroup libpgmodeler_ui
+\class GqbCoreWidget
+\brief Implements a graphical query builder.
+*/
+
+#ifndef GQBCOREWIDGET_H
+#define GQBCOREWIDGET_H
+
+#include "ui_gqbcorewidget.h"
+#include "pgmodelerns.h"
+#include "modelwidget.h"
+#include "sourcecodewidget.h"
+#include "basetable.h"
+#include <QWidget>
+
+class GqbCoreWidget: public QWidget, public Ui::GqbCoreWidget {
```

Comment:
I think ```QueryBuilderCoreWidget``` is a better name, no? I always try to avoid abbreviations on class names. Yeah, I know... I don't have a code guidelines for this project :)

Reply:
sure!